deploy/helm/omni/values.yaml

```
@@ -0,0 +1,68 @@
+
+nameOverride: omni  # optional, if ommitted the Chart name will be used
+debug: false
+domainName: omni.makomedia.nl
+accountUuid: D5C2A21E-7162-4122-B923-A87ED7827F5B
```

Same as domainName...
```suggestion
accountUuid:
```